app/admin/forms.py

```diff
@@ -1,35 +1,61 @@
 from flask_wtf import FlaskForm as Form
-from wtforms import FileField
-from wtforms.validators import (
-    DataRequired,
-)
+from flask_wtf.file import FileField, FileRequired
+from wtforms import ValidationError
+from app.provider.models import Provider
+
+CSV_SCHEMA = None
+DB_SCHEMA = [item for item in dir(Provider)
```

Comment:
This may allow too many things for our purposes I'd recommend explicitly defining the schema. For example the method names that we define in the Provider class.

And while at that just define the CSV Schema instead of db schema and then use it to check against the CSV file.